habitat_sim/utils/filesystem.py

```diff
@@ -0,0 +1,11 @@
+import os
+
+
+def search_dir_tree_for_ext(mydir, ext):
```

Comment:
This can be done with `glob.glob(f"*.{ext}")` -- can also be done recursively like `glob.glob(f"**/*.{ext}", recursive=True)`

Reply:
I ended up not using this function in my next PR so I got rid of it